Formula/arangodb.rb

```diff
@@ -71,11 +72,26 @@ def install
       end
 
       system "cmake", "..", *args
-      system "make", "V=1", "Verbose=1", "VERBOSE=1", "install"
+      system "make", "install"
+
+      %w[arangod arango-dfdb arangosh foxx-manager].each do |f|
+        inreplace etc/"arangodb3/#{f}.conf", pkgshare, opt_pkgshare
+      end
     end
   end
 
   def post_install
+    oldpath_prefix = "#{HOMEBREW_PREFIX}/Cellar/arangodb/3.0."
+    oldpath_regexp = /#{Regexp.escape(oldpath_prefix)}[12]/
+
+    %w[arangod arango-dfdb arangosh foxx-manager].each do |f|
+      file = etc/"arangodb3/#{f}.conf"
+
+      if oldpath_regexp.match(File.read(file))
```

Comment:
you can do
```
inreplace etc/"arangodb3/#{f}.conf" do |s|
  s.gsub! oldpath_regexp, opt_prefix, false
end
```

The `false` sets audit = false so that you don't have to redundantly pre-verify that the inreplace will be successful.